wordpress-mobile/WordPress-iOS
# Switch Zendesk to use encrypted logging

**To test:**
- Send a test ticket to ZenDesk
- Ensure that it contains an encrypted log UUID
- Ensure that the encrypted log can be fetched from `mc`

PR submission checklist:
